packages/react/src/Blankslate/Blankslate.tsx

```diff
@@ -124,6 +127,18 @@ const BlankslateContainerQuery = `
 `
 
 function Blankslate({border, children, narrow, spacious}: BlankslateProps) {
+  const enabled = useFeatureFlag('primer_react_css_modules')
+
+  if (enabled) {
+    return (
+      <div className={classes.Container}>
+        <div className={classes.Blankslate} data-border={border} data-narrow={narrow} data-spacious={spacious}>
+          {children}
+        </div>
+      </div>
+    )
+  }
```

Comment:
_Note for feature flags only_: Idk how to feel about 2 completely different blocks. On one hand, it's very clear that one uses classNames without the Styled(component)Blankslate. On the other hand, if StyledBlankslate accepted sx, we wouldn't be able to do this and would need to merge them in some way.

I think this code looks good and we should ship it + we'll need to migrate a couple other components to see what our practices should be

Reply:
Agreed, this one really feels like a one off because of how simple it is. In reality I think we'll have more mix-and-match and toggling inline